wurst/lib/Stackables.wurst

```diff
@@ -2,6 +2,9 @@ package Stackables
 import StackNSplit
 import ID
 
+init
+    initStackables()
+    
 function initStackables()
     makeItemStackable(ITEM_ACID_BOMB, 2, 1)
```

Comment:
Can these not just be inferred? Is there not a runtime call to get the max charges of an item?

Reply:
@jonathanmclaus even if there is we actually override it with these methods

Reply:
You're not overriding it if you're providing the same value. You're just introducing unnecessary magic numbers. Also, I don't believe it would be an override - you at least have to set it to a smaller value because the max charges of the item isn't being changed.

Reply:
@jonathanmclaus in any case there isn't a way to get that property. People used to "hack" it by setting the item level to the max charges, but frankly I don't really like this approach. Also, that would require hiding a lot of our data in the binary, which I'm also opposed to.

Reply:
OK, good to just confirm that. Not too worried because this is trivial to sync once object definitions are in Wurst.